hugegraph-api/src/main/java/com/baidu/hugegraph/api/graph/BatchAPI.java

```diff
@@ -71,4 +79,82 @@ public BatchAPI() {
             batchWriteThreads.decrementAndGet();
         }
     }
+
+    @JsonIgnoreProperties(value = {"type"})
+    protected static abstract class JsonElement implements Checkable {
+
+        @JsonProperty("id")
+        public Id id;
```

Comment:
use object instead, otherwise it will throw error as follows
`Can not construct instance of com.baidu.hugegraph.backend.id.Id: abstract types either need to be mapped to concrete types`

Reply:
replace Id with Object